goals/operationalUsecases.md

```diff
@@ -0,0 +1,59 @@
+2. # **Manage Projects**
+    **2.1 Add a new self-provisioned namespace**
+As a user, I should be able to create a self-provisioned namespace in the Developer view by Navigating to Home → Projects and by clicking on ‘Create Project’.
+    **2.2 Deactivate a project**
+As a PI of the project, I should be able to remove any project and release the resources associated with it back to the pool.
+    **2.3 Add/remove a project as a PI**
+As a PI, I should be able to create an OpenShift namespace associated with my identity as its PI in ColdFront.
+   **2.4 Add/remove a people as a PI**
+As a PI, I should be able to add additional PIs/people to my project. As a PI, I should be able to remove PIs/people from my project.
+3. ## **Manage Quota**
+    **3.1 Set and modify quotas for projects**
+As a PI of the project, I should be able to request to set and modify compute, storage, external addresses, and object count quotas for the projects I own.
+4. ## **Managing Authorization Policy**
+    **4.1 View and Manage Role bindings**
+As an administrator of the cluster, I should be able to create, view and manage role bindings for the users in the cluster.
+5. ## **Documentation**
+    **5.1 Online Documentation**
+As an administrator or a user, I should be able to perform any routine operations by referring to online documentation which lists the steps that need to be taken to complete an operation.
+6. ## **Hardware Management**
+    **6.1 Add/Remove and track hardware**
+As an administrator of the cluster, I should be able to add/remove cluster nodes. I should also be able to view all the nodes and their status.
+    **6.2 Track faulty hardware**
+As an administrator of the cluster, I should be able to view and track the list of faulty nodes that need to be replaced.
+7. ## **Upgrade**
+    **7.1 Establish OpenShift cluster upgrade process**
+As an administrator of the cluster, I should be able to follow a set of documented instructions to help me in upgrading to newer versions of OpenShift. The document should also establish the process to mitigate any issues that might arise during the upgrade.
+8. ## **Monitoring**
+    **8.1 Generate and share operations alerts**
+    - As an administrator of the cluster, I should be able to generate, view, and share operation alerts in OpenShift using the monitoring tools available in the NERC cluster and store them for 30 days.
+    - Critical alerts should be automatically shared by being published to a Slack channel identified by NERC.
+    - Security events and alerts should be logged and stored for 90 days.
+    - Events and alerts should be archived for researchers' use.
+    **8.2 Logging for OpenShift**
```

Comment:
double-check this when you make it a numbered list because currently, this does not start a new line.